pages/treeherder.py

```diff
@@ -78,6 +84,11 @@ def filter_by(self, term):
         el.send_keys(Keys.RETURN)
         self.wait.until(lambda s: self.result_sets)
 
+    def filter_unclassified_jobs(self):
+        self.find_element(*self._unclassified_failure_filter_locator).click()
+        self.wait.until(lambda s: len(self.result_sets) == 10)
```

Comment:
This wait is not specific to the action of filtering unclassified jobs. If you first click on show next 50 results, then filter unclassified jobs, then you'll have more than 10 result sets.